This patch makes available config file for the Hack set of fonts (as does Solus),
and modify the preferences in 60-latin.conf
diff -rNu fontconfig-2.15.0/conf.d/45-hack.conf fontconfig-2.15.0.new/conf.d/45-hack.conf
--- fontconfig-2.15.0/conf.d/45-hack.conf	1970-01-01 01:00:00.000000000 +0100
+++ fontconfig-2.15.0.new/conf.d/45-hack.conf	2024-11-25 21:26:07.370595000 +0100
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
+<fontconfig>
+  <!-- Declare Hack a monospace font -->
+  <alias>
+    <family>Hack</family>
+    <default><family>monospace</family></default>
+  </alias>
+  <!-- if this file is put in user’s configuration, unset sans-serif family -->
+  <match>
+    <test compare="eq" name="family">
+        <string>Hack</string>
+    </test>
+    <test compare="eq" name="family">
+        <string>sans-serif</string>
+    </test>
+    <edit mode="delete" name="family"/>
+  </match>
+</fontconfig>
diff -rNu fontconfig-2.15.0/conf.d/60-latin.conf fontconfig-2.15.0.new/conf.d/60-latin.conf
--- fontconfig-2.15.0/conf.d/60-latin.conf	2023-12-22 14:43:03.000000000 +0100
+++ fontconfig-2.15.0.new/conf.d/60-latin.conf	2024-11-25 22:20:55.013830171 +0100
@@ -35,8 +35,10 @@
 	<alias>
 		<family>monospace</family>
 		<prefer>
-			<family>Noto Sans Mono</family>
+			<family>Hack</family>
 			<family>DejaVu Sans Mono</family>
+			<family>Noto Mono</family>
+			<family>Noto Sans Mono</family>
 			<family>Inconsolata</family>
 			<family>Andale Mono</family>
 			<family>Courier New</family>
diff -rNu fontconfig-2.15.0/conf.d/meson.build fontconfig-2.15.0.new/conf.d/meson.build
--- fontconfig-2.15.0/conf.d/meson.build	2023-12-22 14:43:03.000000000 +0100
+++ fontconfig-2.15.0.new/conf.d/meson.build	2024-11-25 22:56:31.015982512 +0100
@@ -24,6 +24,7 @@
   '30-metric-aliases.conf',
   '40-nonlatin.conf',
   '45-generic.conf',
+  '45-hack.conf',
   '45-latin.conf',
   '48-spacing.conf',
   '49-sansserif.conf',
@@ -54,6 +57,7 @@
   '30-metric-aliases.conf',
   '40-nonlatin.conf',
   '45-generic.conf',
+  '45-hack.conf',
   '45-latin.conf',
   '48-spacing.conf',
   '49-sansserif.conf',
